src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -113,6 +117,15 @@
 				<!-- New Pending Invitations dialog -->
 				<InvitationHandler v-if="pendingInvitationsCount" ref="invitationHandler" />
 			</div>
+			<TransitionWrapper class="conversations__filters"
+				name="zoom"
+				tag="div"
+				group>
+				<NcChip v-for="filter in filters"
+					:key="filter"
+					:text="t('spreed', filter)"
```

Comment:
this does not work with the translation extraction. you have to have `t('spreed', 'Unread')` and `t('spreed', 'Mentions')` in the source code, otherwise the strings are not listed on transifex

Reply:
better to have some `getFilterLabel` function, where we can also put some comments for translators

Reply:
> this does not work with the translation extraction. you have to have `t('spreed', 'Unread')` and `t('spreed', 'Mentions')` in the source code, otherwise the strings are not listed on transifex

Are you sure?

We use it in many places in many apps.

Reply:
In 100+ places in Talk

Reply:
![image](https://github.com/user-attachments/assets/5e818f34-024a-4caf-8401-f83f5dd8d62f)

Reply:
Oops